src/System/PosixCompat/Files.hsc

```diff
@@ -132,6 +134,7 @@ import System.IO (IOMode(..), openFile, hFileSize, hSetFileSize, hClose)
 import System.IO.Error
 import System.PosixCompat.Types
 import System.Win32.File hiding (getFileType)
+import System.Win32.Time
```

Comment:
Done. I am short on time I will make the `System.Win32.File` imports explicit another time :)